server/app/mutations/grid_certificates/request_certificate.rb

```diff
@@ -119,6 +121,15 @@ def le_client
       @le_client ||= acme_client(self.grid)
     end
 
+    ##
+    # @param [Certificate]
+    def refresh_grid_services(certificate)
+      certificate.grid.grid_services.where(:'certificates.subject' => certificate.subject).each do |grid_service|
```

Comment:
Bonus points for a `certificate.services` method on the model.

(same applies to the `GridSecrest::Common#refresh_grid_services`... #2755 added a `secret.services` method)

BTW what's the difference between `grid_services.where(:'secrets.secret' => secret.name)` and `grid_services.where(:secrets.elem_match => { secret: self.name })`?